File: py/hw/max_min4.py
#date: 10/4/2022.
#description:
	#input: 5 ints.
	#output: print largest, smallest, & mean of the inputted #'s.
#get input from usr..
num1=int(input("Please enter in first number: "))
num2=int(input("Please enter in second number: "))
num3=int(input("Please enter in third number: "))
num4=int(input("Please enter in fourth number: "))
num5=int(input("Please enter in fifth number: "))

#print('')

#print largest.
large=num1

if large<num2:
        large=num2
if large<num3:
        large=num3
if large<num4:
        large=num4
if large<num5:
        large=num5

print("The largest number is:",large)

#print smallest.
small=num1

if small>num2:
        small=num2
if small>num3:
        small=num3
if small>num4:
        small=num4
if small>num5:
        small=num5

print("The smallest number is:",small)

#print mean(∑/n; n=len.(5)).
mean=(num1+num2+num3+num4+num5)/5

print("The mean of the 5 numbers is:",mean)

File: java/hw/MaxMin4.java
/*
 * 5/16/2023
 * input:
 * 	-input 5 integers.
 * output:
 * 	-max,min,mean.
 * due: 5/17/2023
 */
import java.util.Scanner;

public class MaxMin4{
	/*public static String fmt(double d){
	    if(d==(long)d)
	        return String.format("%d",(long)d);
	    else
	        return String.format("%s",d);
	}*/
	public static void main(String[] args){
		//input manager.
		Scanner i1=new Scanner(System.in);

		//get #'s.
		System.out.print("Please enter in first number: ");int n1=i1.nextInt();
		System.out.print("Please enter in second number: ");int n2=i1.nextInt();
		System.out.print("Please enter in third number: ");int n3=i1.nextInt();
		System.out.print("Please enter in fourth number: ");int n4=i1.nextInt();
		System.out.print("Please enter in fifth number: ");int n5=i1.nextInt();

		//get max..
		int max=n1;
		if(max<n2){max=n2;}
		if(max<n3){max=n3;}
		if(max<n4){max=n4;}
		if(max<n5){max=n5;}

		//get min..
		int min=n1;
		if(min>n2){min=n2;}
		if(min>n3){min=n3;}
		if(min>n4){min=n4;}
		if(min>n5){min=n5;}

		//get mean.
		float mean=((float)(n1+n2+n3+n4+n5))/5;

		//display output.
		System.out.printf("The largest number is: %d%n",max);
		System.out.printf("The smallest number is: %d%n",min);
		System.out.printf("The mean of the 5 numbers is: %.1f",mean);

		i1.close();
	}
}